deployment-examples/gcp/k8s/README.md

```diff
@@ -0,0 +1,42 @@
+# Deployment
+
+You can utilize the available bash scripts which is capable of creating a Google Kubernetes Engine cluster and deploying the app into it.
+
+> In the deployment script, the [cron-job](https://github.com/commercetools/k8s-charts/tree/master/charts/cronjob) helm chart is used.
+
+## Prerequisites
+
+- [gcloud sdk](https://cloud.google.com/sdk/docs/install)
+- [docker](https://docs.docker.com/get-docker/)
+- [helm](https://helm.sh/docs/intro/install/)
+- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+
+## Configurations
+
+### Environment variables
+
+| Name                  | Description                                                                                                                    | Default                          |
+| --------------------- | ------------------------------------------------------------------------------------------------------------------------------ | -------------------------------- |
+| `GCLOUD_PROJECT_ID`   | Google Cloud project ID                                                                                                        | `professionalserviceslabs`       |
```

Comment:
Let's not add `professionalserviceslabs` to the OS project. Make it required with no default value.

Reply:
https://github.com/commercetools/commercetools-subscriptions/pull/22/commits/5147220b5a6aa3b69deb2232018824920f3ddf08